### ISSUE TITLE
Glossary Redux

### ISSUE THREAD

**Issue by chenopis** [kubernetes/website#4629]
## TL;DR

This project will standardize Kubernetes terms (as well as their definitions and other metadata), so that their content can be reused consistently throughout the K8s docs (e.g. in concepts, diagrams, etc.). [[project proposal](https://docs.google.com/document/d/1Ar4ploza6zA1JF3YO4e0lzq1RRBWWx8cAZtRQMMEdsw/edit)] [[project tracker](https://github.com/kubernetes/kubernetes.github.io/projects/4)]

## Tasks

- Create a `/_includes/glossary/` directory for snippets of glossary terminology that can be reused throughout the documentation. [done]
- Docs for a topic that match a glossary term should use the glossary snippet as the summary at the top of the doc.
- There may be a complete list and/or smaller curated lists of glossary terminology for specific levels of understanding. 
- Docs tying together concepts using the glossary terminology should have tooltips that use the snippets.
- Interactive diagrams that use any of the glossary terms should also use the snippets.

**Comment by chenopis** [comment]
cc @abiogenesis-now

**Comment by chenopis** [comment]
This is related to https://github.com/kubernetes/kubernetes.github.io/issues/1191

**Comment by chenopis** [comment]
@heidecke We need to get you on the road to being a sig-docs-maintainer.

**Comment by abiogenesis-now** [comment]
@heidecke I had an idea about kubernetes visualizations---not sure how feasible so I'll think about it more and discuss with @chenopis tomorrow. If it seems reasonable can I make an addition to the bottom of your google doc?

**Comment by heidecke** [comment]
@abiogenesis-now Definitely, feel free to add/comment.

**Comment by chenopis** [comment]
This has been completed and is now live: https://kubernetes.io/docs/reference/glossary/?fundamental=true
